.eslintrc-fluid.json

```diff
@@ -20,12 +20,7 @@
             "error",
             "all"
         ],
-        "dot-notation": [
-            "error",
-            {
-                "allowKeywords": false
-            }
-        ],
+        "dot-notation": [2, { "allowKeywords": true, "allowPattern": "catch" }],
```

Comment:
Just so I understand, in my reading of the docs it sounds like the `allowPattern` option controls what's allowed in bracket notation.  Was that done on purpose to allow the older `promise["catch"]` style?  What happens if you omit it?

Reply:
Also, `true` is the default, we can probably just remove the options block entirely if we don't need the pattern.

Reply:
Yes, you're right I can omit the allowPattern block as redundant. And I guess as a result the entire dot-notation rule since the default for allowKeywords is true - thanks for the catch!

Reply:
Sure, the pull looks fine to me, let me know if you cut a release and I can whip out a dependency update release for `fluid-lint-all`.